### PR TITLE
Adds some HippieStation related tips

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: strings/tips.txt [hunk 1]
```diff
@@ -186,3 +186,13 @@ As a Ghost, you can double click on people, bots, or the singularity to follow t
...
+Vampires which have not yet reached full power will catch on fire if they ingest holy water or if they are in the chapel!
+Vampires can heal by laying inside coffins!
+The only way vampire bats can exist is if they are summoned by a Vampire!
+Shadowlings are able to enthrall mindshielded people, but it takes much longer to do so.
+Shadowlings are dangerous foes to face alone! When fighting Shadowlings, bring others!
+As a Vampire, you are able to heal stamina damage with Rejuvenate, and later that ability will heal normal damage, too!
+As a Vampire, you can convert normal people into Vampires with Lillith's Pact. Be warned, the new vampire is NOT loyal to you in any way!
+As a Shadowling, be aware, Veil does not work on flares, glowsticks, or most other non-electric lights!
+As a Shadowling, you should help thralls if they call for help, as long as it does not put yourself in danger!
+If you see a Cluwne or Catbeast, kill it to death!
```
Comment (ghost) [inline strings/tips.txt:198]: This is not a vampire related tip mate

Comment (ghost) [reply strings/tips.txt:198]: Title fixed

**Comment by JohnGinnane** [comment]
@HippieStation/head does this need votes? If so can we get some?

**Review by ghost — APPROVED** [review]
Should be fine, we can always revert

**Pull request merged**